app/assets/stylesheets/components/_accordions.scss

```diff
@@ -7,3 +7,29 @@
   background-image: asset-url('plus-blue.png');
   background-image: asset-url('plus-blue.svg');
 }
+
+.helpful-links-accordion {
+  margin-top: 1rem;
+
+  .usa-accordion-button {
+    background-color: $color-gray-cool-light;
+    background-image: asset-url('uswds/angle-arrow-up-primary-hover.png');
```

Comment:
Minor style thing: Use double quotes here instead of single quotes. Functionally similar in SCSS, but consistent with our established formatting.